### PR TITLE
Starting to implement the template for ttt10

### PR DESCRIPTION
**Pull request by jose-luis-rs**
Template for the implementation of the TTT10 silicon detectors

---

Checklist:

* [x] Rebased against `dev` branch
* [x] My name is in the resp. CONTRIBUTORS/AUTHORS file
* [x] Followed the [pull request guidlines](https://opensource.creativecommons.org/contributing-code/pr-guidelines/) and the [Git workflow](https://github.com/AnarManafov/GitWorkflow/blob/master/GitWorkflow.markdown)
* [x] Followed the [seven rules of great commit messages](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

File: tttx/pars/R3BTttxContFact.cxx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    // Creates the Container objects with all accepted contexts and adds them to
+    // the list of containers for the TTTX library.
+
+    FairContainer* p1 = new FairContainer("tttxMappingPar", "TTTX Mapping Parameters", "TttxMappingParContext");
+    p1->addContext("TttxMappingParContext");
+    containers->Add(p1);
+
+    FairContainer* p2 = new FairContainer("tttxStripCalPar", "TTTX StripCal Parameters", "TttxCalParContext");
+    p2->addContext("TttxCalParContext");
+    containers->Add(p2);
```
Comment (ryotani) [inline tttx/pars/R3BTttxContFact.cxx]: We don't have a mapping par, but would have cal2hit par. Could to define p1 as stripcalpar and p2 as hitpar?

**Comment by ryotani** [comment]
Thanks. They look reasonable and I can add changes on this commit.

File: tttx/pars/R3BTttxContFact.cxx (hunk 1 above)
Comment (jose-luis-rs) [reply tttx/pars/R3BTttxContFact.cxx]: I am still working on it, so I will add the class for Cal2Hit

Comment (jose-luis-rs) [reply tttx/pars/R3BTttxContFact.cxx]: Hello @ryotani 
Which parameters do you need to move data from Cal to Hit?

Comment (ryotani) [reply tttx/pars/R3BTttxContFact.cxx]: @jose-luis-rs For HitPar I think parameters as
- NumDetPar (default = 2)
- NumZReconstructPar (default = 2)
- ZReconstructPar (TArrayF with fNumDet * fNumZRecPar elements)

Also for the CalPar, if these parameters can be added that's great.
- NumStrips
- StripsInUse (TArrayF with fNumDet * fNumStrips elements)

Comment (jose-luis-rs) [reply tttx/pars/R3BTttxContFact.cxx]: I will do it today

Comment (jose-luis-rs) [reply tttx/pars/R3BTttxContFact.cxx]: @ryotani, let me know if you are happy with this to merge the code. Thanks

**Comment by ryotani** [comment]
@jose-luis-rs This PR is fine with me at least for the ttt10 side.

**Pull request merged**